src/transformers/audio_utils.py

```diff
@@ -363,6 +379,8 @@ def spectrogram(
         db_range (`float`, *optional*):
             Sets the maximum dynamic range in decibels. For example, if `db_range = 80`, the difference between the
             peak value and the smallest value will never be more than 80 dB. Must be greater than zero.
+        remove_dc_offset (`bool`, *optional*):
```

Comment:
Good for me since the argument name matches that used in `torchaudio.fbank`: https://github.com/pytorch/audio/blob/4bbf65e44e5730d4b7deba745ba11375c1665db8/torchaudio/compliance/kaldi.py#L165